docs/Python-API.md

```diff
@@ -64,7 +64,7 @@ env = UnityEnvironment(file_name="3DBall", base_port=5005, seed=1, side_channels
 - `worker_id` indicates which port to use for communication with the
   environment. For use in parallel training regimes such as A3C.
 - `seed` indicates the seed to use when generating random numbers during the
-  training process. In environments which do not involve physics calculations,
```

Comment:
"no physics" does not imply "deterministic"

Reply:
Agree. We called out physics before though just to let people know that the physics is non-determinstic. So just because you don't use a `Random` call in your code, it doesn't mean you are guaranteed deterministic.

Reply:
The inverse of the statement is also not true: physics simulations are not inherently non-deterministic (although it sounds like Unity's implementation is).

Reply:
Yes 😄